metals/src/main/scala/scala/meta/internal/metals/codelenses/SuperMethodCodeLens.scala

```diff
@@ -31,10 +31,8 @@ final class SuperMethodCodeLens(
     val textDocument = textDocumentWithPath.textDocument
     val path = textDocumentWithPath.filePath
 
-    val search = implementationProvider.defaultSymbolSearchMemoize(
-      path,
-      textDocument
-    )
+    def search(query: String) = textDocument.symbols.find(_.symbol == query)
```

Comment:
<del> I'm just wondering if we do this, we can't find the parent symbol who comes from another file (semanticdb), or class files 🤔 </del>
↑ Nevermind, `search` method is used just for finding the symbol on the cursor and finding overriddenSymbol is done by
https://github.com/scalameta/metals/blob/1404b26ae810f387ab784feea8b492d03c19de9a/metals/src/main/scala/scala/meta/internal/metals/codelenses/SuperMethodCodeLens.scala#L59